### ISSUE TITLE
Annotation import was unsuccessful. For YOLO labels to be loaded correctly, labels.txt file is required.

### ISSUE THREAD

**Issue by Xianyi-Wang** [SkalskiP/make-sense#314]
**Describe the bug**
I used MakeSense and exported the annotations to YOLO format, but when I uploaded these image files again and imported these annotations, it fails with error message as follow:
<img width="515" alt="image" src="https://user-images.githubusercontent.com/82689228/223616318-1913807d-2e63-4d95-a636-40a119ccfe91.png">
I guess this is because my file name is a timestamp and contains multiple dots, like "1670577347.002843.txt", but I saw that the bug was fixed in issue https://github.com/SkalskiP/make-sense/issues/140 , I don't know why it still doesn't work.
I'm a newbie, please pardon me if I make some mistakes.

**To Reproduce**
1. select Import Annotations
2. select Multiple Files in YOLO Format
3. Select annotation files in format "1670577347.002843.txt"
4. Import fails with error "Annotation import was unsuccessful. For YOLO labels to be loaded correctly, labels.txt file is required."

**Expected behavior**
I think it should success.

**Screenshots**
<img width="943" alt="image" src="https://user-images.githubusercontent.com/82689228/223617091-abe1052e-bf7a-454f-a57a-3f4577c60028.png">

**Desktop (please complete the following information):**
 - OS: MacOS 13.2.1 
 - Browser: chrome
 - Version: MakeSense web version

**Comment by Xianyi-Wang** [comment]
I have changed both the image and label filenames to ensure they don't  contain multiple "." but it still doesn't work. 
And I have also tried to git clone and run make-sense locally, but importing still failed.

**Comment by SkalskiP** [comment]
Hi, @Xianyi-Wang 👋🏻! Did you provided `labels.txt` file?

**Comment by Xianyi-Wang** [comment]
> 

Yes, I did provided annotation files in txt format, but importing failed.

**Comment by tapas** [comment]
I am also facing the same problem in importing annotations, I am unsure what format in labels.txt file is needed. Please resolve the issue

**Comment by PowZone** [comment]
Same here. I need to export in Yolo format but there isn't a labels.txt in the output zip.

Thx!

**Comment by Nauman691** [comment]
i want to export the files in yolo format but there are no yolo format files in the downloaded folder

**Comment by rickkk856** [comment]
Any progress on this since then? Also need got this error,

**Comment by Halvani** [comment]
Same here! Only the annotations are exported, but not the labels.txt itself. This is very annoying if you have created a large number of labels. From the annotation text files you will only able to extract the IDs of the labels, but not the strings they are assigned to...

**Comment by rickkk856** [comment]
> ame here! Only the annotations are exported, but not the labels.txt itself. This is very annoying if you have created a large number of labels. From the annotation text files you will only able to extract the IDs of the labels, but not the strings they are assigned to...

I've managed to solve the issue by creating the labels.txt file manually and putting it in the same folder of the files that i had previoulsy exported....

My suggestion is when starting a project make the labels.txt file like this for example

```
0 Class_01
1 Class_02
3 Class_03
```

And then create these labels in the same order at the web app, so when you export them the labels.txt file will not be exported but you'll have a backup made manually just to import them back and continue labelling in another day.

**Comment by yuxin7** [comment]
> **Describe the bug** I used MakeSense and exported the annotations to YOLO format, but when I uploaded these image files again and imported these annotations, it fails with error message as follow: <img alt="image" width="515" src="https://user-images.githubusercontent.com/82689228/223616318-1913807d-2e63-4d95-a636-40a119ccfe91.png"> I guess this is because my file name is a timestamp and contains multiple dots, like "1670577347.002843.txt", but I saw that the bug was fixed in issue #140 , I don't know why it still doesn't work. I'm a newbie, please pardon me if I make some mistakes.
> 
> **To Reproduce** Steps to reproduce the behavior:
> 
> 1. select Import Annotations
> 2. select Multiple Files in YOLO Format
> 3. Select annotation files in format "1670577347.002843.txt"
> 4. Import fails with error "Annotation import was unsuccessful. For YOLO labels to be loaded correctly, labels.txt file is required."
> 
> **Expected behavior** I think it should success.
> 
> **Screenshots** <img alt="image" width="943" src="https://user-images.githubusercontent.com/82689228/223617091-abe1052e-bf7a-454f-a57a-3f4577c60028.png">
> 
> **Desktop (please complete the following information):**
> 
> * OS: MacOS 13.2.1
> * Browser: chrome
> * Version: MakeSense web version


您好。您这个问题最后解决了吗？？？

**Comment by Hassan-Tamer** [comment]
It didn't work for me when I had the labels in a file with "classes.txt" as a name but when I renamed it "labels.txt" It worked successfully

**Comment by yusuphwasiu** [comment]
To resolve this, create a `labels.txt` file in the same folder with the annotations (i.e `.txt`) and put the labels in it as shown in the example below. 

```
0 Dog
1 Cat
2 Fish
```

**Comment by Damarcreative** [comment]
In the collection of .txt annotation files, make sure there is a `labels.txt` file that contains a list of labels. Therefore, you cannot upload only annotation files; You must also upload the `labels.txt` file. Here's an example:
```
labels.txt
image1.txt
image2.txt
image3.txt
image4.txt
```
Make sure to include the labels.txt file along with the annotation file when uploading it to makesense.ai.

**Comment by heroacool** [comment]
> In the collection of .txt annotation files, make sure there is a `labels.txt` file that contains a list of labels. Therefore, you cannot upload only annotation files; You must also upload the `labels.txt` file. Here's an example:
> 
> ```
> labels.txt
> image1.txt
> image2.txt
> image3.txt
> image4.txt
> ```
> 
> Make sure to include the labels.txt file along with the annotation file when uploading it to makesense.ai.

it works. thank you!!!

**Comment by vcangfy** [comment]
> In the collection of .txt annotation files, make sure there is a `labels.txt` file that contains a list of labels. Therefore, you cannot upload only annotation files; You must also upload the `labels.txt` file. Here's an example:在 .txt 注释文件集合中，请确保有一个包含标签列表的 `labels.txt` 文件。因此，您不能仅上传注释文件；您还必须上传 `labels.txt` 文件。以下是示例：
> 
> ```
> labels.txt
> image1.txt
> image2.txt
> image3.txt
> image4.txt
> ```
> 
> Make sure to include the labels.txt file along with the annotation file when uploading it to makesense.ai.将其上传到 makesense.ai 时，请确保将 label.txt 文件与注释文件一起包含在内。

you are right! thanks
